bindings/c/include/manifoldc.h

```diff
@@ -301,8 +301,6 @@ ManifoldRect *manifold_rect_mul(void *mem, ManifoldRect *r, float x, float y);
 int manifold_rect_does_overlap_rect(ManifoldRect *a, ManifoldRect *r);
 int manifold_rect_is_empty(ManifoldRect *r);
 int manifold_rect_is_finite(ManifoldRect *r);
-ManifoldCrossSection *manifold_rect_as_cross_section(void *mem,
-                                                     ManifoldRect *r);
```

Comment:
@geoffder I also did a refactor here to move Rect into public like Box, because I wanted to use it without depending on cross_section. The only real change is that I had to remove `AsCrossSection`. Doesn't seem like a big loss, since there's already a `CrossSection(Rect)` constructor. FYI in case this has wider implications.